deltacat/compute/compactor_v2/constants.py

```diff
@@ -92,3 +92,16 @@
 SHA1_HASHING_FOR_MEMORY_OPTIMIZATION_DISABLED = env_bool(
     "SHA1_HASHING_FOR_MEMORY_OPTIMIZATION_DISABLED", False
 )
+
+# Whether to enable logging if any partition is found
+# to be non-compliant with the bucketing spec.
+ENABLE_BUCKETING_SPEC_COMPLIANCE_LOGGING = env_bool(
```

Comment:
Any reason to not default this to `True`? Cost perhaps?

Reply:
Yes, cost.